### PR TITLE
Remove some unnecessary variables from the hostgroups functional test

### PR DESCRIPTION
**Pull request by skottler**

### CODE REVIEW

**Comment by skottler** [comment]
[test]

**Comment by skottler** [comment]
I think this should be fine and just remove some unnecessary vars unless `assert_response :ok` relies on the retrieval of the JSON that takes place in the `template` variable.

File: test/functional/hostgroups_controller_test.rb [hunk 1]
```diff
@@ -72,7 +71,6 @@ def test_update_valid
   def test_update_valid_json
     Hostgroup.any_instance.stubs(:valid?).returns(true)
     put :update, {:format => "json", :id => Hostgroup.first, :hostgroup => {}}, set_session_user
-    template = ActiveSupport::JSON.decode(@response.body)
```
Comment (domcleal) [inline test/functional/hostgroups_controller_test.rb:75]: I'd guess by the test name that this is designed to parse the JSON to ensure it's vaild, not just check the response is successful.  If you're trying to eliminate warnings, I'd retain the decode but remove the unused variable.

**Comment by skottler** [comment]
@domcleal yeah I think you're right. Updated to just remove the variable assignment.

**Comment by skottler** [comment]
[test]

**Comment by domcleal** [comment]
Merged as 926f04e, thanks.

**Pull request closed without merging**